src/getMessageAuthor.ts

```diff
@@ -88,3 +119,40 @@ function authorFromGitHubContext(): MessageAuthor | null {
     icon_url: sender.avatar_url
   }
 }
+
+async function getSenderFromPRMerger(
+  octokit: OctokitClient
+): Promise<MessageAuthor> {
+  const payload = context.payload
+
+  const matches = (payload.head_commit?.message as string).match(/\(#(\d+)\)$/)
+
+  if (!matches) {
+    throw new Error(
+      `Failed to parse PR number from commit message: '${payload.head_commit?.message}'.`
+    )
+  }
+
+  const prNumber = Number(matches[1])
+
+  if (!payload.repository) {
+    throw new Error('WebhookPayload does not include repository information')
+  }
+
+  const mergedBy = (
+    await octokit.rest.pulls.get({
+      owner: payload.repository.organization,
+      repo: payload.repository.name,
+      pull_number: prNumber
+    })
+  ).data.merged_by
```

Comment:
I think this should work:

```suggestion
  const mergedBy = (
    await octokit.rest.pulls.get({
      ...context.repo,
      pull_number: prNumber
    })
  ).data.merged_by
```